website/docs/cloud-docs/workspaces/health.mdx

```diff
@@ -60,6 +61,19 @@ Terraform Cloud pauses health assessments if the latest run ended in an errored
 
 Health assessments may take longer to complete when you enable health assessments in many workspaces at once or your workspace contains a complex configuration with many resources.
 
+### On Demand Assessments
+
+-> **Note:** On Demand Assessments are only available through browser sessions. If the workspace requirements are not satisfied, you should see one or more error messages on the page indicating the blockers to enable health assessments. The button will be disabled until the requirements are satisfied.
```

Comment:
```suggestion
-> **Note:** On demand assessments are only available in the Terraform Cloud web UI. You can only trigger an on demand assessment if the workspace satisfies all assessment prerequisites.
```

can we say web UI instead of browser sessions? also simplified the language a little bit.

Reply:
yes I think Web UI should be fine.